catroidTest/src/org/catrobat/catroid/uitest/util/UiTestUtils.java

```diff
@@ -2132,4 +2133,12 @@ public boolean isSatisfied() {
 
 		return wantedState;
 	}
+
+	public static void leaveStageSafeButSlow(Solo solo) {
+		if(solo.getCurrentActivity().getClass() == StageActivity.class){
```

Comment:
Yea the `instanceof` operator would be also an option, but in this case i favour comparing classes. ([instanceof vs. getClass( )](http://stackoverflow.com/questions/4989818/instanceof-vs-getclass)) :ocean: